js/background.js

```diff
@@ -10,6 +10,7 @@ var fullscreenSetInterval;
 var defaultConfig = {
     frequency: 28,
     length: 2,
+    postpone: 3,
```

Comment:
I haven't had chance to test this yet, but I think that the settings are persistent across version updates, which means that when a user receives this update, config.postpone would be undefined.

I think we need to update https://github.com/LukaszWiktor/breaktimer/blob/7a68d93d830874353e0c76a28c541ee6032188e4/js/background.js#L52 to something like

```
// Grab config from local storage or take defaultConfig
chrome.storage.local.get('config', function(data) {
    if (typeof(data.config) === 'undefined') {
        setConfig(defaultConfig);
    } else {
        if (typeof(data.config.postpone) === 'undefined') {
            data.config.postpone = defaultConfig.postpone;
        }
        setConfig(data.config);
    }
});
```

Not ideal, but I can't think of a better way of doing this.